pkg/graveler/graveler.go

```diff
@@ -1932,6 +1995,16 @@ func (g *KVGraveler) stagingEmpty(ctx context.Context, repositoryID RepositoryID
 	return true, nil
 }
 
+func (g *KVGraveler) sealedEmpty(ctx context.Context, repositoryID RepositoryID, repo *Repository, branch *Branch) (bool, error) {
+	itrs, err := g.listSealedTokens(ctx, branch)
+	if err != nil {
+		return false, err
+	}
+	it := NewCombinedIterator(itrs...)
```

Comment:
make sure there is len(iters)>0 for panic - consider returning an empty value iterator - where nothing is returned and no fear of panic in runtime

Reply:
done